dashboards-observability/public/components/event_analytics/explorer/visualizations/config_panel/config_panes/config_controls/data_configurations_panel.tsx

```diff
@@ -194,7 +194,7 @@ export const DataConfigPanelItem = ({
             },
           })
         );
-        await fetchData(false);
```

Comment:
Question: what was this false an argument for before its removal??

Reply:
fetchData function had an optional parameter as isRefresh: boolean. We had added it to fix an issue and later removed it as it was not required any more. However, this call remained with an argument for isRefresh, hence we cleaned it up.